src/internal/converters/ics/ics.go

```diff
@@ -338,7 +338,7 @@ func updateEventProperties(ctx context.Context, event models.Eventable, iCalEven
 
 	// STATUS - https://www.rfc-editor.org/rfc/rfc5545#section-3.8.1.11
 	cancelled := event.GetIsCancelled()
-	if cancelled != nil {
+	if cancelled != nil && ptr.Val(cancelled) {
```

Comment:
nit: Should we make this check a bit more explict?
```suggestion
	if cancelled != nil && ptr.Val(cancelled) == true {
```

Reply:
I think this should be fine.